tests/ui/single_char_push_str.fixed

```diff
@@ -12,4 +18,6 @@ fn main() {
     string.push('\x52');
     string.push('\u{0052}');
     string.push('a');
+
+    get_string!().push_str("ö");
```

Comment:
This one should be also modified to be a call to `push`, good thing you used this character, I think we hit a bug :)

This line:
https://github.com/rust-lang/rust-clippy/blob/d88b9b7543756afef81dcb2e2c37a11603d513d5/clippy_lints/src/methods/mod.rs#L3236

Should read:
```rust
        if string.chars().count() == 1;
```

Because that char is two bytes, see [len docs](https://doc.rust-lang.org/std/string/struct.String.html#method.len).

Reply:
Ah good point! I added the fix.